### PR TITLE
World generation

### PR DESCRIPTION
**Pull request by Jakllp**
Creating a world with all dungeons and even a spawnpoint

### CODE REVIEW

File: resources/helper/God.gd [hunk 1]
```diff
@@ -0,0 +1,208 @@
+extends Node
```
Comment (Thyrexx) [inline resources/helper/God.gd:1]: Why Node?

File: resources/helper/PerlinHelper.gd [hunk 2]
```diff
@@ -0,0 +1,17 @@
+extends Node
```
Comment (Thyrexx) [inline resources/helper/PerlinHelper.gd:1]: Why Node?

File: resources/helper/PosHelper.gd [hunk 3]
```diff
@@ -0,0 +1,148 @@
+extends Node
```
Comment (Thyrexx) [inline resources/helper/PosHelper.gd:1]: Why Node?

File: resources/helper/StructurePlacer.gd [hunk 4]
```diff
@@ -0,0 +1,97 @@
+extends Node
```
Comment (Thyrexx) [inline resources/helper/StructurePlacer.gd:1]: Why Node?

Comment (Jakllp) [reply resources/helper/StructurePlacer.gd:1]: Bc godot is dumb sometimes and defaulted to that again on my PC

File: resources/helper/PosHelper.gd (hunk 3 above)
Comment (Jakllp) [reply resources/helper/PosHelper.gd:1]: Bc godot is dumb sometimes and defaulted to that again on my PC

File: resources/helper/PerlinHelper.gd (hunk 2 above)
Comment (Jakllp) [reply resources/helper/PerlinHelper.gd:1]: Bc godot is dumb sometimes and defaulted to that again on my PC

File: resources/helper/God.gd (hunk 1 above)
Comment (Jakllp) [reply resources/helper/God.gd:1]: Bc godot is dumb sometimes and defaulted to that again on my PC

**Comment by Jakllp** [comment]
You missed one xD

**Pull request merged**